### ISSUE TITLE
F1 is too low?

### ISSUE THREAD

**Issue by yin-hong** [changzhisun/entrel-joint-mrt#3]
Hello!
I have run your project on NYT dataset. But I found F1 is too low in the early running while prediction is larger than recall. Is this normal? And how many epoches F1 can reach the comparaly value of paper?

Thank you!

**Comment by changzhisun** [comment]
> Hello!
> I have run your project on NYT dataset. But I found F1 is too low in the early running while prediction is larger than recall. Is this normal? And how many epoches F1 can reach the comparaly value of paper?
> 
> Thank you!
I need to check the log file to answer your question. However,
recently, there have been some problems on our server, 
which lead to the failure of login.
I am so sorry about it, and I'll give you a reply later.

**Comment by changzhisun** [comment]
I need to check the log file to answer your question. However,
recently, there have been some problems on our server,
which lead to the failure of login.
I am so sorry about it, and I'll give you a reply later.

**Comment by yin-hong** [comment]
> I need to check the log file to answer your question. However,
> recently, there have been some problems on our server,
> which lead to the failure of login.
> I am so sorry about it, and I'll give you a reply later.

Oh, thank you.I am very grateful and look forward to your answer.

**Comment by changzhisun** [comment]
> > I need to check the log file to answer your question. However,
> > recently, there have been some problems on our server,
> > which lead to the failure of login.
> > I am so sorry about it, and I'll give you a reply later.
> 
> Oh, thank you.I am very grateful and look forward to your answer.

Hi, for NYT dataset, I set the number of epochs to 100, and select the best relation performance according to validation set.
In the fine-tune step, I set the number of epochs to 10.
Generally speaking,  in our experiments,  the precision is about 0.6, and the recall is about 0.4.

**Comment by yin-hong** [comment]
> > > I need to check the log file to answer your question. However,
> > > recently, there have been some problems on our server,
> > > which lead to the failure of login.
> > > I am so sorry about it, and I'll give you a reply later.
> > 
> > 
> > Oh, thank you.I am very grateful and look forward to your answer.
> 
> Hi, for NYT dataset, I set the number of epochs to 100, and select the best relation performance according to validation set.
> In the fine-tune step, I set the number of epochs to 10.
> Generally speaking, in our experiments, the precision is about 0.6, and the recall is about 0.4.

Thanks for your answer. Your answer is very helpful to me.I noticed your answer saying use the fine-tune step. I just want to know in the fine-tune step , do you only reduce the learning rate and then train for 10 epochs?
Thank you!

**Comment by changzhisun** [comment]
> > > > I need to check the log file to answer your question. However,
> > > > recently, there have been some problems on our server,
> > > > which lead to the failure of login.
> > > > I am so sorry about it, and I'll give you a reply later.
> > > 
> > > 
> > > Oh, thank you.I am very grateful and look forward to your answer.
> > 
> > 
> > Hi, for NYT dataset, I set the number of epochs to 100, and select the best relation performance according to validation set.
> > In the fine-tune step, I set the number of epochs to 10.
> > Generally speaking, in our experiments, the precision is about 0.6, and the recall is about 0.4.
> 
> Thanks for your answer. Your answer is very helpful to me.I noticed your answer saying use the fine-tune step. I just want to know in the fine-tune step , do you only reduce the learning rate and then train for 10 epochs?
> Thank you!

In pre-training step, we minimize L_{ent} + L{rel}. In fine-tuning step, we minimize L_{ent} + L_{rel} + L_{mrt}. The learning rate in the fine-tune is default value.

**Comment by yin-hong** [comment]
> > > > > I need to check the log file to answer your question. However,
> > > > > recently, there have been some problems on our server,
> > > > > which lead to the failure of login.
> > > > > I am so sorry about it, and I'll give you a reply later.
> > > > 
> > > > 
> > > > Oh, thank you.I am very grateful and look forward to your answer.
> > > 
> > > 
> > > Hi, for NYT dataset, I set the number of epochs to 100, and select the best relation performance according to validation set.
> > > In the fine-tune step, I set the number of epochs to 10.
> > > Generally speaking, in our experiments, the precision is about 0.6, and the recall is about 0.4.
> > 
> > 
> > Thanks for your answer. Your answer is very helpful to me.I noticed your answer saying use the fine-tune step. I just want to know in the fine-tune step , do you only reduce the learning rate and then train for 10 epochs?
> > Thank you!
> 
> In pre-training step, we minimize L_{ent} + L{rel}. In fine-tuning step, we minimize L_{ent} + L_{rel} + L_{mrt}. The learning rate in the fine-tune is default value.

Thank you very much!  God bless you!
